tests/karate/src/test/java/fe/create_lao.feature

```diff
@@ -0,0 +1,25 @@
+@env=android,web
+Feature: Create LAO
+
+  Background: Driver basic setup
+    * def page_object = 'classpath:fe/utils/<env>.feature@name=basic_setup'
```

Comment:
A comment here maybe explaining what is going about the karate tags system

Reply:
If you think so, I can add a link and a short explaination yes.

Reply:
This should be clearer